### PR TITLE
simplify prow  test scripts, export KUBECONFIG

### PR DESCRIPTION
**Pull request by rkpagadala**

### CODE REVIEW

**Comment by ldemailly** [comment]
/test istio-unit-tests
/lgtm
(without looking in depth, red and less copy pasta looks great - just expecting you carefully checked/will check the builds)

**Comment by rkpagadala** [comment]
fixes #3474

File: prow/istio-postsubmit.sh [hunk 1]
```diff
@@ -29,29 +29,11 @@ set -u
...
-  GIT_SHA="$(git rev-parse --verify HEAD)"
-fi
-cd $ROOT
+source ${ROOT}/prow/lib.sh
+setup_and_export_git_sha
 
-# Build
-${ROOT}/bin/init.sh
+cd $ROOT
+make init
```
Comment (mattdelco) [inline prow/istio-postsubmit.sh]: I'm not clear on the current state of dependencies, but there can be a perf benefit to batching together the calls to make.  E.g., the "make init" could be part of the next call to make, such as:
<pre>time make init localTestEnv test</pre>
or possibly even combine all 3 together
<pre>HUB="gcr.io/istio-testing"
TAG="${GIT_SHA}"
time ISTIO_DOCKER_HUB="${HUB}" make init localTestEnv test push HUB="${HUB}" TAG="${TAG}"
</pre>

This is more an observation than a request, and I'm not sure if there's be much of a benefit now.  I had opted for robust depedency checking that's likely favor this kind of approach, but other changes were made to take shortcuts to benefit the split builds in CircleCI and I'm not sure what the end result is like.

**Pull request merged**